posts/2024-02-16-Clippy-deprecating-feature-cargo-clippy.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: post
+title: "Clippy: Deprecating `feature = \"cargo-clippy\"`"
+author: The Clippy Team
+release: true
+---
+
+Since Clippy [`v0.0.97`] and before it was shipped with `rustup`, Clippy
+implicitly added a `feature = "cargo-clippy"`[^1] when linting your code with
+`cargo clippy`.
+
+[^1]: It's likely, that you didn't even know that Clippy implicitly sets this
+    feature. This is intentional, as we stopped advertising and documenting this
+    a long time ago.
```

Comment:
Maybe clarify that it was not a Cargo feature, but a config that impersonnated Cargo features.
```suggestion
[^1]: It's likely, that you didn't even know that Clippy implicitly sets this
    config (which was not a Cargo feature). This is intentional, as we stopped advertising and documenting this
    a long time ago.
```